lib/cisco_node_utils/cmd_ref/vdc.yaml

```diff
@@ -7,8 +7,8 @@ _exclude: [N3k, N5k, N6k, N9k]
 
 _template:
   get_command: 'show run vdc all'
-  get_context: ['/^vdc <vdc>/']
-  set_context: ['vdc <vdc>']
+  get_context: ['(?)/^vdc <vdc>/']
```

Comment:
As I stated in the email thread - for most of the properties in the feature, this is not optional - it would be preferable to explicitly override the context for properties that it doesn't apply to, rather than making this template change.